app/assets/javascripts/slices/app/helpers/i18n.js

```diff
@@ -0,0 +1,5 @@
+Handlebars.registerHelper('i18n', function(options) {
```

Comment:
I wonder if this could have a clearer name? at first glance I though this helper would return a boolean as that's what `slices.i18n` is